### ISSUE TITLE
keyboard navigation does not work on tab menu

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#486]
### Describe the bug

Keyboard navigation does not work for the tab menu.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/9bfecb9g?file=src%2Fapp%2Ftabs-tabmenu-demo.ts

### Environment

Primeng: 19.1.2

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

Please press <space> go to the next tab with the right arrow and press enter. The navigation did not run.

### Expected behavior

The keyboard navigation should work as standard tabs: the navigation should start when the enter button is pressed on the new tab.

> Migrated from `primefaces/primeng#18321` — originally by `@jma` on 2025-05-13

**Comment by geromegrignon** [comment]
*[Originally posted by `@Retch` on 2025-05-14T09:51:03.000Z]*

Should be the same like [this](https://github.com/primefaces/primeng/issues/17774)

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-11-11T00:20:37.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
